bundle.Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG quay_expiration=never
+ARG release_tag=0.0.0
+
+FROM quay.io/operator-framework/operator-sdk:v1.13 as builder
```

Comment:
Operator-sdk is up to v1.14.0. Should we be using that?

Reply:
Bumped up to 1.14.0